### PR TITLE
Add ManyUniqueTablesSimulation

### PR DESCRIPTION
**Pull request by dimas-b**
This simulation attempts to represent a scenario where
multiple users create unique tables in the main branch.
    
The main purpose of this simulation is exploratory and
resilience testing of Nessie servers.

### CODE REVIEW

File: perftest/simulations/src/test/scala/org/projectnessie/perftest/gatling/ManyUniqueTablesSimulation.scala [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  * This simulation is not very realistic from the end user perspective because
+  * using a new table UUID is each commit is not the intended way of using the
+  * Nessie Catalog. It is generally expected that table UUIDs are reused and
+  * each table is mentioned in more than one commit.
+  *
+  * The goal of this simulation is mostly to probe the robustness and
+  * scalability of the handling of global data in Nessie under contention when
+  * many table UUIDs are at play.
+  *
+  * This simulation is meant to run in two major modes: <ul> <li>Using a high
```
Comment (tomekl007) [inline perftest/simulations/src/test/scala/org/projectnessie/perftest/gatling/ManyUniqueTablesSimulation.scala:39]: can we add two pre-defined configurations per those two scenarios?
I see that we have this:
```
object ConcurrentCommitsParams {
  def fromSystemProperties(): ConcurrentCommitsParams = {
    val base = BaseParams.fromSystemProperties()
    ConcurrentCommitsParams(
      Integer.getInteger("sim.warmup.seconds", 1).toInt,
      Integer.getInteger("sim.rampup.seconds", 1).toInt,
      Integer.getInteger("sim.plateau.seconds", 1).toInt,
      Integer.getInteger("sim.warmup.rate", 1).toInt,
      base.opRate,
      base.note
    )
  }
```
but it uses `1` as default. We could create two specific configs: `capacityScenario` and `responseTimes` with a defaults that make sense for testing those.

Comment (dimas-b) [reply perftest/simulations/src/test/scala/org/projectnessie/perftest/gatling/ManyUniqueTablesSimulation.scala:39]: let me think about this :hourglass:

Comment (dimas-b) [reply perftest/simulations/src/test/scala/org/projectnessie/perftest/gatling/ManyUniqueTablesSimulation.scala]: I've refactored param classes a bit and added scaladoc... hope it clarifies the use cases :)

**Comment by dimas-b** [comment]
will revisit after the WIP model changes are merged... if this scenario is still relevant.

**Pull request closed without merging**